src/app.py

```diff
@@ -20,37 +21,58 @@ def setup_logging(logger_name):
     logging.basicConfig(level=logging.DEBUG)
 
 
-def main(generated_files_folder='/usr/src/tmp',
+@click.command()
+@click.option('--vcf', is_flag=True, help='generates VCF files')
+@click.option('--ortho', is_flag=True, help='generates orthology files')
+@click.option('--daf', is_flag=True, help='generates DAF files')
+@click.option('--expr', is_flag=True, help='generates expression files')
+def main(vcf, ortho, daf, expr,
+         generated_files_folder=os.path.abspath(os.path.join(os.getcwd(), os.pardir)) + '/output',
          fasta_sequences_folder='sequences',
          skip_chromosomes={'Unmapped_Scaffold_8_D1580_D1567'}):
-    #generate_vcf_files(generated_files_folder, fasta_sequences_folder, skip_chromosomes)
-    #generate_orthology_file(generated_files_folder, alliance_db_version)
-    #generate_daf_file(generated_files_folder, alliance_db_version)
-    generate_expression_file(generated_files_folder, alliance_db_version)
+
+    print('INFO\t\tFiles output: ' + generated_files_folder)
+
+    if vcf is True:
+        print('INFO\t\tGenerating VCF files')
+        generate_vcf_files(generated_files_folder, fasta_sequences_folder, skip_chromosomes)
+    if ortho is True:
+        print('INFO\t\tGenerating Orthology files')
+        generate_orthology_file(generated_files_folder, alliance_db_version)
+    if daf is True:
+        print('INFO\t\tGenerating DAF files')
+        generate_daf_file(generated_files_folder, alliance_db_version)
+    if expr is True:
+        print('INFO\t\tGenerating Expression files')
+        generate_expression_file(generated_files_folder, alliance_db_version)
 
 
 def generate_vcf_files(generated_files_folder, fasta_sequences_folder, skip_chromosomes):
     os.makedirs(generated_files_folder, exist_ok=True)
     os.makedirs(fasta_sequences_folder, exist_ok=True)
     variants_query = """MATCH (s:Species)-[:FROM_SPECIES]-(a:Allele)-[:VARIATION]-(v:Variant)-[l:LOCATED_ON]-(c:Chromosome)
-MATCH (v:Variant)-[:VARIATION_TYPE]-(st:SOTerm)
-OPTIONAL MATCH (a:Allele)-[:IS_ALLELE_OF]-(g:Gene)
-RETURN c.primaryKey AS chromosome,
-       v.globalId AS globalId,
-       right(v.paddingLeft,1) AS paddingLeft,
-       v.genomicReferenceSequence AS genomicReferenceSequence,
-       v.genomicVariantSequence AS genomicVariantSequence,
-       v.hgvsNomenclature AS hgvsNomenclature,
-       v.dataProvider AS dataProvider,
-       a.symbol AS symbol,
-       collect(a.primaryKey) AS alleles,
-       CASE WHEN g IS NOT NULL THEN collect(g.primaryKey) ELSE [] END AS alleleOfGenes,
-       l.start AS start,
-       l.end AS end,
-       l.assembly AS assembly,
-       s.name AS species,
-       st.nameKey AS soTerm
-    """
+                     MATCH (v:Variant)-[:VARIATION_TYPE]-(st:SOTerm)
+                     MATCH (v:Variant)-[:ASSOCIATION]-(p:GenomicLocation)
```

Comment:
When you have a second match line instead of saying MATCH again you should use a comma. The query will end up running slightly faster

Reply:
OK, I will change that.